CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 ### Added
+- Added `Visualize` function to parse the graph in a container into DOT format
```

Comment:
What is DOT format exactly?

Reply:
We can clarify that in the changelog. How about,

    Added `Visualize` function to visualize the state of the container in the
    GraphViz DOT format.

Reply:
I've updated the changelog to provide more information!